docs/output.md

```diff
@@ -151,39 +183,52 @@ The output format of this command is an edge list in CSV format.
 
 </details>
 
-This step uses the `pixelator single-cell graph` command.
-The input is the edge list dataframe (CSV) generated in the collapse step and after filtering it
-by count (`--graph_min_count`), the connected components of the graph (graphs) are computed and
-added to the edge list in a column called "component".
+### Cell-calling, filtering, and annotation
 
-The graph command has the option to recover components (technical multiplets) into smaller
-components using community detection to find and remove problematic edges.
-(See `--multiplet_recovery`). The information to keep track of the original and
-newly recovered components are stored in a file (components_recovered.csv).
+This step uses the `pixelator single-cell annotate` command.
 
-### Cell-calling, filtering, and annotation
+The annotate command takes as input the edge list file generated in the graph command. It parses, and filters the
+edgelist to find putative cells, and it will generate a PXL file containing the edgelist, and an
```

Comment:
```suggestion
molecules grouped by "component" ID to find putative cells, and it will generate a PXL file containing the edges of the graphs in an edgelist, and an
```